fearless/Modules/Staking/StakingUnbondConfirm/StakingUnbondConfirmInteractor.swift

```diff
@@ -0,0 +1,241 @@
+import UIKit
+import SoraKeystore
+import RobinHood
+import BigInt
+import FearlessUtils
+
+final class StakingUnbondConfirmInteractor: RuntimeConstantFetching, AccountFetching {
+    weak var presenter: StakingUnbondConfirmInteractorOutputProtocol!
+
+    let singleValueProviderFactory: SingleValueProviderFactoryProtocol
+    let substrateProviderFactory: SubstrateDataProviderFactoryProtocol
+    let settings: SettingsManagerProtocol
+    let runtimeService: RuntimeCodingServiceProtocol
+    let operationManager: OperationManagerProtocol
+    let extrinsicServiceFactory: ExtrinsicServiceFactoryProtocol
+    let feeProxy: ExtrinsicFeeProxyProtocol
+    let accountRepository: AnyDataProviderRepository<AccountItem>
+    let chain: Chain
+    let assetId: WalletAssetId
```

Comment:
Looks like everything here should be private

Reply:
We just need a rule for this. In my opinion it is convenient to have public properties which just were passed as parameters in init.